src/i18n/de_DE/buildings.ts

```diff
@@ -1669,4 +1685,45 @@ export default {
         schools: [],
         icon: 'poo',
     },
+    24: {
+        caption: 'Reiterstaffel',
+        color: '#227722',
+        coins: 50,
+        credits: 300_000,
+        levelPrices: {
+            credits: [],
+            coins: [],
+        },
+        extensions: [
+            ...multiplyExtension(
+                {
+                    caption: 'Reiterstaffel',
+                    credits: 300_000,
+                    coins: 25,
+                    duration: '5 Tage',
+                    isVehicleExtension: true,
+                    parkingLotReservations: [
+                        [134, 135, 136],
+                        [134, 135, 136],
+                        [134, 135, 136],
+                        [137],
+                        [137],
+                        [137],
+                    ],
+                    givesParkingLots: 6,
+                },
+                6
+            ),
+        ],
+        levelcost: ['Keine Ausbaustufen möglich'],
+        maxBuildings: 'Keine Grenze',
+        maxLevel: 0,
+        special: '',
+        startPersonnel: 8,
+        startParkingLots: 6,
+        startVehicles: [],
+        schoolingTypes: ['Polizei'],
+        schools: [8],
+        icon: 'house-medical',
```

Comment:
don't we want smth with a horse or horses as icon?

Reply:
https://fontawesome.com/icons/horse ? There is no good icon for that :smile:

Reply:
yeah, I think that is an okayish icon :)